CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+
+<a id='changelog-0.5.0.0'></a>
+## 0.5.0.0 — 2023-02-17
+
+First release of `lmdb-simple` fork. See
```

Comment:
I think this should have some kind of subheader that indicates this is a breaking change?

Reply:
I've now added all noteworthy changes that we (the Consensus team) have made.